sphinxext/photofinish/svgtopng.py

```diff
@@ -343,350 +343,3 @@ def __exit__(self, exc, val, trb) -> bool:
         raise NoToolError("\n".join(err_msg))
 
     raise FailedConversionError("\n".join(err_msg + log))
-
-
-import gc
-import subprocess
-import sys
-import traceback
-from contextlib import suppress
-from dataclasses import dataclass
-from pathlib import Path
-from shutil import which
-from typing import List, Optional, Union
-
-
-class NoToolError(RuntimeError):
-    pass
-
-
-class FailedConversionError(RuntimeError):
-    pass
-
-
-# code from https://utcc.utoronto.ca/~cks/space/blog/python/GetAllObjects
-# Changes:
-# - nest `_getr`
-# - Swap dict for set
-# - Make function a generator
-def _all_python_objects():
-    """Return a list of all live Python
-    objects, not including the list itself."""
-
-    # Recursively expand slist's objects
-    # into olist, using seen to track
-    # already processed objects.
-    def _getr(slist, seen):
-        for e in slist:
-            if id(e) in seen:
-                continue
-            seen.add(id(e))
-            yield e
-            tl = gc.get_referents(e)
-            if tl:
-                yield from _getr(tl, seen)
-
-    gcl = gc.get_objects()
-    seen = set()
-    # Just in case:
-    seen.add(id(gcl))
-    seen.add(id(seen))
-    # _getr does the real work.
-    yield from _getr(gcl, seen)
-
-
-def get_sphinx() -> Optional["Sphinx"]:
-    with suppress(Exception):
-        if "sphinx" in sys.modules or any("sphinx" in arg for arg in sys.argv):
-            # This code is probably in a sphinx build
-            from sphinx.application import Sphinx
-
-            for obj in _all_python_objects():
-                if isinstance(obj, Sphinx):
-                    return obj
-    return None
-
-
-SPHINX_INKSCAPE_CONVERTER_BIN = None
-SPHINX_RSVG_CONVERTER_BIN = None
-SPHINX_FOUND = False
-
-
-def svg_to_png(
```

Comment:
This function is actually the one that gets called. Is the other function actually identical?

Reply:
Yes.